kong/plugins/acl/hooks.lua

```diff
@@ -1,26 +0,0 @@
-local events = require "kong.core.events"
```

Comment:
We now handle all invalidation events that are easily deducible (when the entity being invalidated is the same as the one being updated, or when it is a foreign entity that we can track because of our schema's `foreign` property) via a new property on the schema, which is `cache_key`. When an entity, or a parent entity is being updated, we compute its cache_key via its schema, and tell all the other Kong nodes "that's the unique cache key you need to invalidate".